app/src/test/java/com/example/concordia_campus_guide/ModelTests/RouteTest.java

```diff
@@ -61,6 +61,17 @@ public void getMainTransportTypeTest() {
         assertEquals(expectedMainTransportType, route.getMainTransportType());
     }
 
+    @Test
+    public void getSetSteps() {
+        // Arrange
+        List<TransportType> expectedSteps = new ArrayList<>();
+        Route route = new Route("20 mins", "Autoroute 15-S", "driving");
+        route.setSteps(expectedSteps);
+
+        // Act & Assert
+        assertEquals(expectedSteps, route.getSteps());
+    }
```

Comment:
minor suggestion, but usually we add the word "Test" at the end of the method name

Reply:
oh yeah I normally do that but forgot to do it fo rthat one, thanks!